corehq/apps/data_dictionary/util.py

```diff
@@ -382,29 +382,25 @@ def is_case_type_or_prop_name_valid(case_prop_name):
     return match_obj is not None
 
 
-@quickcache(['domain'], timeout=24 * 60)
-def used_case_props_by_domain(domain):
-    case_prop_agg = NestedAggregation('case_props', CASE_PROPERTIES_PATH).aggregation(
-        TermsAggregation('props', PROPERTY_KEY)
-    )
-    query = (
-        CaseSearchES()
-        .domain(domain)
-        .size(0)
-        .aggregation(case_prop_agg)
+@quickcache(vary_on=['domain'], timeout=60 * 60)
+def get_used_props_by_case_type(domain):
+    agg = TermsAggregation('case_types', 'type.exact').aggregation(
+        NestedAggregation('case_props', CASE_PROPERTIES_PATH).aggregation(
```

Comment:
Just confirming: Should this not be 'case_properties', like `CASE_PROPERTIES_AGG` in [this file](https://github.com/dimagi/commcare-hq/blob/e06f12981391b33b227b1baf13a7533934106949/corehq/apps/geospatial/es.py)

Reply:
That is correct, it's just the name of the aggregation which we use to access it.